server/src/pages/shot/view.js

```diff
@@ -346,13 +346,15 @@ class Body extends React.Component {
     let editButton;
     const highlight = this.state.highlightEditButton ? <div className="edit-highlight" onClick={ this.onClickEdit.bind(this) } onMouseOver={ this.onMouseOverHighlight.bind(this) } onMouseOut={ this.onMouseOutHighlight.bind(this) }></div> : null;
 
-    const activeFavClass = this.props.expireTime ? "" : "is-fav";
+    const activeFavClass = this.props.expireTime ? "un-fav" : "is-fav";
```

Comment:
`un-fav` only affects the span and `is-fav` only affects the div.  It's confusing when class names are unnecessarily added to elements.  Why not have the styles of `un-fav` be the default and then `is-fav` to toggle?  Then we don't even need to change this line.